Guide/src/dev_guide/getting_started/build_openvmm.md

```diff
@@ -88,3 +88,26 @@ Note: the specific package that throws this error may vary, and may not always b
 You attempted to build OpenVMM without first restoring necessary packages.
 
 Please run `cargo xflowey restore-packages`, and try again.
+
+### use of unstable library feature 'absolute_path'
```

Comment:
Lets avoid tying the troubleshooting header to any particular feature, so we can keep the same header well into the future

```suggestion
### use of unstable library feature
```